app/models/in_person_enrollment.rb

```diff
@@ -2,11 +2,12 @@ class InPersonEnrollment < ApplicationRecord
   belongs_to :user
   belongs_to :profile
   enum status: {
-    pending: 0,
-    passed: 1,
-    failed: 2,
-    expired: 3,
-    canceled: 4,
+    establishing: 0,
+    pending: 1,
+    passed: 2,
+    failed: 3,
+    expired: 4,
+    canceled: 5,
```

Comment:
In general I think renumbering enums like this is dangerous.... since we haven't launched yet this is probably fine. I think it would preferable to do `establishing: 7` in the future

Reply:
Due to it not being in production we won't break anything also establishing is the phase that comes before pending so logically that order makes sense